cartography/intel/aws/ec2.py

```diff
@@ -847,6 +888,50 @@ def load_ec2_vpcs(neo4j_session, data, region, current_aws_account_id, aws_updat
         )
 
 
+TAG_RESOURCE_TYPE_MAPPINGS = {
+    'instance': {'label': 'EC2Instance', 'property': 'instanceid'},
+    'network-interface': {'label': 'NetworkInterface', 'property': 'id'},
+    'security-group': {'label': 'EC2SecurityGroup', 'property': 'id'},
+    'subnet': {'label': 'EC2Subnet', 'property': 'subnetid'},
+    'vpc': {'label': 'AWSVPC', 'property': 'id'},
+}
+
+TAG_INGEST_TEMPLATE = Template("""
+MATCH (n:$label{$property: {id}})
+WITH n
+MERGE (tag:Tag{key: {Key}})
+ON CREATE SET tag.firstseen = timestamp()
+WITH tag, n
+MERGE (n)<-[r:TAGGED]-(tag)
+ON CREATE SET r.firstseen = timestamp()
+SET r.value = {Value}, r.lastupdated = {aws_update_tag}
```

Comment:
I might be missing it, but where is `{Key}` set?

What is a way to uniquely identify tags?  If an ARN or similar doesn't exist in AWS then we might need to create a fake one like in [GCP](https://github.com/lyft/cartography/blob/1148c98e58238a83833a4a20f620fa49df1bf009/cartography/intel/gcp/compute.py#L184).

Reply:
One more related note: we should create an index for `AWSTag` and decide on what field to use for it.